vendor/BUILD.bazel

```diff
@@ -234,3 +235,28 @@ cc_library(
         "nunicode-gen",
     ],
 )
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+cmake(
```

Comment:
Let's discuss here, about user cc_library to replace foreign_cc.cmake.
Using foreign_cc.cmake can let us directly use the libraries' shipped compiling setting, but it will cost too much time on compiling. I didn't do the final test, but I do run raw cmake build, the cost is under 100s with raw cmake on my end but will cost 400s+ when using foreign_cc.cmake.

The con for cc_library is we must compile the library on our own, but I don't think it's an obstacle.